examples/ezusb.c

```diff
@@ -130,7 +130,7 @@ static int ezusb_write(libusb_device_handle *device, const char *label,
 	if (verbose > 1)
 		logerror("%s, addr 0x%08x len %4u (0x%04x)\n", label, addr, (unsigned)len, (unsigned)len);
 	status = libusb_control_transfer(device,
-		LIBUSB_ENDPOINT_OUT | LIBUSB_REQUEST_TYPE_VENDOR | LIBUSB_RECIPIENT_DEVICE,
+	(uint8_t)LIBUSB_ENDPOINT_OUT | (uint8_t)LIBUSB_REQUEST_TYPE_VENDOR | (uint8_t)LIBUSB_RECIPIENT_DEVICE,
```

Comment:
@tormodvolden @mcuee @seanm I had to add those ugly "(uint_8)" in several places to make latest VS compiler happy, as discussed here:
https://github.com/libusb/libusb/issues/1649 
https://github.com/libusb/libusb/pull/1650
https://github.com/libusb/libusb/pull/1655

Shall I:
a) keep them in this PR ? And fix them with another PR aimed at fix this issue 
b) remove them
c) wait for https://github.com/libusb/libusb/issues/1649 to be resolved / merged and rebase this one, removing them de facto

The point is that if I remove them and do nothing else, people won't be able to compile with a decently recent VS, kind of defeating the introduction of a new feature targeting Windows...